documentation/asciidoc/computers/linux_kernel/about-kernel.adoc

```diff
@@ -15,9 +15,14 @@ There are many reasons you may want to put something into the kernel:
 * You've fixed a generic kernel bug
 * You've fixed a Raspberry Pi-specific kernel bug
 
-Initially, you should fork the https://github.com/raspberrypi/linux[Linux repository] and clone that on your build system; this can be either on your Raspberry Pi or on a Linux machine you're using for cross-compiling. You can then make your changes, test them, and commit them into your fork.
+==== Contribute to the Linux kernel
```

Comment:
The previous version had a (brief) explanation of when you'd submit to "upstream Linux" and when you'd submit to "Raspberry Pi Linux", but that distinction seems to have disappeared now?